docs/stack/client/howToUseClient.md

```diff
@@ -0,0 +1,203 @@
+# How To Use
+
+In the following guide we describe how to instantiate classes and discuss the common methods shared by all Entities. Please refer to [complete API reference](https://daostack.github.io/client/docs/globals.html) for detailed list of properties and methods
+
+## Configuration: Arc
+
+The `Arc` class holds the basic configuration and serves as the main entrypoint when using the library. The user of the library must provide some basic configuration options.
+
+### Configuration Options
+
+  Please see the [API Reference](https://daostack.github.io/client/docs/classes/arc.html#constructor) for details of Configuration Options and their types.
+
+  - _**contractInfos[]**_: contracts details. Can be set/fetch using subgraph by `setContractInfos`/`fetchContractInfos`
+  - _**graphqlHttpProvider**_: http connection to the subgraph of TheGraph protocol
+  - _**graphqlPrefetchHook()**_: function executed before sending graphQL query
+  - _**graphqlSubscribeToQueries**_: determines if query should subscribe to updates from the graphProvider. Default True
+  - _**graphqlWsProvider**_: web socket connection to subgraph of TheGraph protocol
+  - _**ipfsProvider**_: connection to ipfs provider which is used as the data storage layer by DAOstack
+  - _**web3Provider**_: connection to ethereum node
+  - _**web3ProviderRead**_: connection to ethereum node to read Arc data. If provided arc will read all data from this provider, else set same as `web3Provider`
+
+### Example
+
+```
+import { Arc } from '@daostack/client'
+
+const arc = new Arc({
+  graphqlHttpProvider: "https://subgraph.daostack.io/subgraphs/name/v23",
+  graphqlWsProvider: "wss://ws.subgraph.daostack.io/subgraphs/name/v23",
+  web3Provider: `wss://mainnet.infura.io/ws/v3/e0cdf3bfda9b468fa908aa6ab03d5ba2`,
+  ipfsProvider: {
+    "host": "subgraph.daostack.io",
+    "port": "443",
+    "protocol": "https",
+    "api-path": "/ipfs/api/v0/"
+  }
+})
+
+// before we can use the Arc instance to send transactions, we need to provide it
+// with information on where the contracts can be found
+// query the subgraph for the contract addresses, and use those
+await arc.fetchContractInfos()
+```
+
+**Note:**
+
+Some of these configuration settings are _**optional**_ to use `@daostack/client`. Like
+
+  - For creating and sending transactions to the blockchain, it is sufficient to provide the `web3Provider`.
+  - For fetching data from the subgraph, the `web3` and `ipfs` providers can be omitted when the library is only used.
+
+## Entities
+
```

Comment:
They encapsulate graphql queries so you don't have to worry about graphql (though you can also use graphql queries, using the static method `arc.apolloClient` as explained [where?]).